cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -2,20 +2,24 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   labels:
-    application: skipper-ingress-redis
+    application: skipper-ingress
+    component: redis
     version: v6.2.4
   name: skipper-ingress-redis
   namespace: kube-system
 spec:
   replicas: {{ .ConfigItems.skipper_redis_replicas }}
+  podManagementPolicy: Parallel
   selector:
     matchLabels:
-      application: skipper-ingress-redis
+      application: skipper-ingress
+      component: redis
```

Comment:
If you change this anyway, you may want to go with what we did for a bunch of other components where we use the matchLabel: `<resource-type>=<name>` e.g. `statefulset=skipper-ingress-redis`. That let's you decouple `application` and `component` from the selector which is only really about ownership between statefulset and pods and you can freely adjust `application` and `component` should it be needed in the future.

Ref: https://github.com/zalando-incubator/kubernetes-on-aws/pull/4611

Reply:
We also need to check zmon alerts/checks, such that we do not catch by application label all the things